common/test/src/IO/WorldReaderTest.cpp

```diff
@@ -47,6 +48,26 @@ namespace TrenchBroom {
             return nullptr;
         }
 
+        inline void
```

Comment:
Our function declarations usually all go into one line, so return type and `inline` should not get an extra line. Or am I now contradicting what you were saying @ericwa ?